### PR TITLE
Add coffee thoughts

### PR DESCRIPTION
**Pull request by davidklaw**
Adds random coffee messages when the :coffee: pot is ready.

### CODE REVIEW

File: scripts/coffeepot.coffee [hunk 1]
```diff
@@ -15,20 +15,33 @@
...
 
+coffee_thoughts = [
+  "(coffee) (successful) — Coffee is Ready!",
+  "(coffee) (successful) — (klaw)",
+  "(coffee) (successful) — Keep Calm and Drink Coffee",
+  "(coffee) (successful) — Life Begins After Coffee",
+  "(coffee) (successful) — The world consumes close to 2.25 billion cups of coffee every day.",
+  "(coffee) (successful) — Coffee is the world's second most valuable traded commodity, only behind petroleum.",
+  "(coffee) (successful) — Coffee is most effective if consumed between 9:30 am and 11:30 am.",
+  "(coffee) (successful) — In Korea and Japan, there is a Cat Cafe where you can go to drink coffee and hang out with cats for hours.",
```
Comment (macklinu) [inline scripts/coffeepot.coffee]: Also in Portland. The dream of cat cafes is alive in Portland.

File: scripts/coffeepot.coffee [hunk 2]
```diff
@@ -49,8 +62,3 @@ module.exports = (robot) ->
       res.end "OK"
     else
       res.end "Nope."
-
```
Comment (macklinu) [inline scripts/coffeepot.coffee]: Thank you.

Comment (davidklaw) [inline scripts/coffeepot.coffee]: :bow:

File: scripts/coffeepot.coffee (hunk 1 above)
Comment (davidklaw) [inline scripts/coffeepot.coffee]: :cat2:

**Pull request merged**